source/about/people/volunteer_moderators.md

```diff
@@ -0,0 +1,30 @@
+# Volunteer Moderators
+
+(Content moderation)[/help/moderation/index.html] ensures that arXiv is a valuable resource for researchers. Submissions are checked that they appear in relevant subject areas and are of likely interest to professional researchers. arXiv relies on hundreds of volunteers, who are subject matter experts, to achieve its mission.
+
+## What moderators do
+
+Moderators provide quick checks on submissions. If a submission looks acceptable, the moderator does nothing and the submission gets posted. The vast majority of submissions meet the requirements set by arXiv and can be posted quickly. Only if there is an issue of classification or scholarly content does the moderator need to take action either to change the classification or indicate an issue.
+
+Moderators are not referees and are not doing detailed checks or providing feedback. A typical day may include 10 papers to screen and may take about 10 minutes of time.
+
+## Benefits of moderation are:
+
+- Keeping up to date with newest research in the field
```

Comment:
To match other bullets: "Keeping up to date with the newest research in the field"